scripts/publish-ghpages.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+set -o errexit -o nounset
+
+# User info
+git config user.name "patternfly-build"
+git config user.email "patternfly-build@redhat.com"
+git config --global push.default simple
+
+## Deploy to github pages
+if [ -d "patternfly.github.io" ]; then
+  rm -rf patternfly.github.io
+fi
+git clone git@github.com:patternfly/patternfly.github.io.git
+rsync -av --delete --exclude .git source/_site/ patternfly.github.io
+cd patternfly.github.io
```

Comment:
Yeah, I've given this some though, and I feel we could accomplish this with a single repo (consolidating on the patternfly.github.io repo).  I'll re-factor my approach and update this PR.